dll/win32/iphlpapi/ifenum_reactos.c

```diff
@@ -149,7 +149,9 @@ BOOL hasArp( HANDLE tcpFile, TDIEntityID *arp_maybe ) {
                               NULL );
     if( !NT_SUCCESS(status) ) return FALSE;
 
-    return (type & AT_ARP);
+    /* This was previously treated incorrectly as a single bitmapped flag.
+     * But this is not correct because AT_ARP is 0x280. */
```

Comment:
Remove the comment.

Reply:
@SergeGautherie, I have no problem with the comment being removed, but I would prefer that the person who commits this do it. That way others will have the benefit of seeing it during the review cycle. Thanks.